ooiservices/app/__init__.py

```diff
@@ -38,6 +38,10 @@ def create_app(config_name):
         env.from_yaml(os.path.join(basedir, 'config.yml'))
     celery.conf.update(BROKER_URL=app.config['REDIS_URL'],
                 CELERY_RESULT_BACKEND=app.config['REDIS_URL'])
+    if app.config['REDIS_CACHE']:
+        cache.config = {'CACHE_TYPE': 'redis', 'CACHE_REDIS_URL': app.config['REDIS_URL']}
+    else:
+        cache.config = {'CACHE_TYPE': 'simple'}
```

Comment:
@oceanzus this isn't the correct way, we aren't giving them the option to choose what cache type they are using.